packages/react-scripts/config/webpack.config.js

```diff
@@ -388,7 +388,7 @@ module.exports = function (webpackEnv) {
             // smaller than specified limit in bytes as data URLs to avoid requests.
             // A missing `test` is equivalent to a match.
             {
-              test: [/\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/],
+              test: [/\.avif$/, /\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/],
```

Comment:
Unfortunately this didn't work correctly 😢 

`image/avif` appears to be missing from [mime-db](https://github.com/jshttp/mime-db) which is used by this loader. This means that when the image is inlined, it doesn't know the correct mime type to use in the data URI, and breaks the image:

![image](https://user-images.githubusercontent.com/6355370/94720734-b15a3480-0309-11eb-9b55-3668e81d4feb.png)

As a temporary fix until the mime type is available, i've updated the webpack config to define loader config specifically for `.avif` files so that we can hardcode the mime type. This worked well in my testing.